deployability/modules/provision/playbooks/package/install/install.j2

```diff
@@ -1,3 +1,7 @@
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
```

Comment:
Remove the copyright header in jinja2 templates, as they will be introduced each time the template is used

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5152/commits/98401a0a222ec7cb65be53b354ca3ad4be4b6210